src/get-correct-gear.ts

```diff
@@ -0,0 +1,20 @@
+export const getCorrectGear = (speed: number) => {
+    if (speed === 0) {
+        return 0;
+    }
+    if (speed <= 25) {
+        return 1;
+    }
+    if (speed >= 25 && speed <= 35) {
+        return 2;
+    }
+    if (speed >= 35 && speed <= 50) {
+        return 3;
+    }
+    if (speed >= 50 && speed <= 65) {
+        return 4;
+    }
+    if (speed >= 65) {
```

Comment:
You don't need this final `if` because you want to return `5` for all other cases. So just return outside any if statement